src/lightning/pytorch/loggers/utilities.py

```diff
@@ -29,12 +29,18 @@ def _version(loggers: List[Any], separator: str = "_") -> Union[int, str]:
     return separator.join(dict.fromkeys(str(logger.version) for logger in loggers))
 
 
-def _scan_checkpoints(checkpoint_callback: Checkpoint, logged_model_time: dict) -> List[Tuple[float, str, float, str]]:
+def _scan_checkpoints(
+    checkpoint_callback: Checkpoint, logged_model_time: dict, include_distributed_checkpoints: bool = False
+) -> List[Tuple[float, str, float, str]]:
     """Return the checkpoints to be logged.
 
     Args:
         checkpoint_callback: Checkpoint callback reference.
         logged_model_time: dictionary containing the logged model times.
+        include_distributed_checkpoints: flag to include distributed directories.
```

Comment:
I suggest not introducing a new argument here. There is already `WandbLogger(log_model=True|False|"all" ...)` which can be used to turn on or off logging the checkpoints.